### PR TITLE
no space before function paren

### PR DESCRIPTION
**Pull request by mparke**

### CODE REVIEW

**Comment by eatonphil** [comment]
Hmm, one point of the linter is to have uniform code. I don't care so much which way it goes, but I do like that it enforces one way. It can get a little nitpicky sometimes, but it's sort of a slippery slope to start disabling checks.

**Comment by mparke** [comment]
we recently re-enabled anon `functions`, which you couldn't write before, this just enables `function() {}` vs `function () {}`

**Comment by eatonphil** [comment]
I understand. My point is that now you can write `function() {}` or `function () {}`: we lose style consistency. (And even then, this obviously isn't the first PR to loosen things up. ) I really don't care as much about any of the linter points except for that the linter enforces some particular style.

File: .eslintrc [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "consistent-return": 0,
     "react/no-multi-comp": 0,
     "react/no-did-mount-set-state": 0,
+    "space-before-function-paren": 0,
```
Comment (abemassry) [inline .eslintrc]:
```diff
-    "space-before-function-paren": 0,
```

**Pull request closed without merging**